flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -671,8 +671,11 @@ static void go_enable_fly_home()
 		vtol_fsm_path_desired.End[2] = -RTH_MIN_ALTITUDE;
 	}
 
-	vtol_fsm_path_desired.StartingVelocity = RTH_VELOCITY;
-	vtol_fsm_path_desired.EndingVelocity = RTH_VELOCITY;
+	float rth_vel = 3.0f;
```

Comment:
I am going to audit and see how we handle this systematically.  I have a feeling we come up short.  I'm kinda inclined to follow the convention and not assign it, but open a separate issue/PR where we make these things fault for now on.